packages/config/test/al-config-client-v2.spec.ts

```diff
@@ -0,0 +1,49 @@
+import {
+  AlDefaultClient,
+  AlLocatorService
+} from '@al/core';
+import { expect } from 'chai';
+import { describe } from 'mocha';
+import {
+  restore,
+  SinonSpy,
+  stub
+} from 'sinon';
+import { ALConfigV2 } from '../src/index';
+
+describe('CARGO CLIENT V2', () => {
+    let spy: SinonSpy;
+    const config = {
+        apiBaseURL: 'https://api.cloudinsight.alertlogic.com',
+        service: 'config',
+        version: 'v2',
+    };
+    const cargoURL = `${config.apiBaseURL}/${config.service}/${config.version}`;
+    const accountId = '2';
+
+    beforeEach(() => {
+      AlLocatorService.setContext( { environment: "production" } );
+      return AlDefaultClient.setGlobalParameters({ noEndpointsResolution: true });
+    });
+    afterEach(() => restore());
+
+    describe('Schedules', () => {
+        describe('When batch delete is called', () => {
+            beforeEach(() => {
+                spy = stub(AlDefaultClient as any, 'axiosRequest').returns(Promise.resolve({status: 204}));
+            });
+
+            it('Should call the batchDeleteSchedules (DELETE).', async () => {
```

Comment:
not testing Delete but listing functionality. please update description accordingly